test/tests/TileElements.cpp

```diff
@@ -82,7 +82,7 @@ TEST_F(TileElementWantsFootpathConnection, SlopedPath)
 TEST_F(TileElementWantsFootpathConnection, Stall)
 {
     // Stalls usually have one path direction flag, but can have multiple (info kiosk for example)
-    const TrackElement* const stallElement = map_get_track_element_at(19 << 5, 15 << 5, 14);
+    const TrackElement* const stallElement = map_get_track_element_at({ 19 << 5, 15 << 5, 14 << 3 });
```

Comment:
This is becoming more ugly with every change ;). Can you put the data into a TileCoordsXYZ and then convert that to CoordsXYZ?

Reply:
Done